kore/src/Control/Monad/Unify.hs

```diff
@@ -0,0 +1,106 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+
+module Control.Monad.Unify where
```

Comment:
I think this should be in the `Kore.Unification` tree because it depends on other things there, perhaps `Kore.Unification.Unifier`?